packages/eslint-plugin/tests/rules/require-await.test.ts

```diff
@@ -338,6 +402,16 @@ async function* foo(): void {
           data: {
             name: "Async generator function 'foo'",
           },
+          suggestions: [
+            {
+              output: `
+function* foo(): void {
```

Comment:
(no action requested) 

Oh... This is a silly test case, since generators _cannot_ have `void` return.....

But, granted the rule flags, I think this is the correct suggestion; just remove the `async` and don't do anything else.